### PR TITLE
Add onClientPlayerWeaponSwitch for remote players

### PR DESCRIPTION
**Pull request by correaAlex**
This PR add onClientPlayerWeaponSwitch for remote players #1141

### CODE REVIEW

File: Shared/sdk/net/rpc_enums.h [hunk 1]
```diff
@@ -104,6 +104,7 @@ enum eElementRPCFunctions
     TAKE_ALL_WEAPONS,
     SET_WEAPON_AMMO,
     SET_WEAPON_SLOT,
+    REMOTE_PLAYER_WEAPON_SWITCH,
```
Comment (ccw808) [inline Shared/sdk/net/rpc_enums.h]:
```
/*
    HEY, YOU!
    ONLY ADD ENUMS TO THE END OF THIS FILE,
    ABOVE `NUM_RPC_FUNCS`.
    THANKS.
*/
```

Comment (correaAlex) [reply Shared/sdk/net/rpc_enums.h]: Oh, sorry 🤦😂

File: Shared/sdk/net/rpc_enums.h [hunk 2]
```diff
@@ -104,7 +104,7 @@ enum eElementRPCFunctions
     TAKE_ALL_WEAPONS,
     SET_WEAPON_AMMO,
     SET_WEAPON_SLOT,
-
+
```
Comment (qaisjp) [inline Shared/sdk/net/rpc_enums.h]:
```suggestion

```

Comment (correaAlex) [reply Shared/sdk/net/rpc_enums.h]: Sorry, but I don't understand why remove the indent. That was before my pull request. And sorry for my poor English

Comment (qaisjp) [reply Shared/sdk/net/rpc_enums.h]: The indent shows up here https://github.com/multitheftauto/mtasa-blue/pull/1143/files, which means it was introduced by your pull request

**Comment by CrosRoad95** [comment]
have you missed to check network version or new packet does not require it?

**Comment by correaAlex** [comment]
> have you missed to check network version or new packet does not require it?

I didn't think about it... Yeah, it has to be done, but I don't really know how to do it right.

**Comment by CrosRoad95** [comment]
https://github.com/multitheftauto/mtasa-blue/wiki/BitStream

**Comment by PlatinMTA** [comment]
> We'd probably need a new event name, as scripts may rely on the event triggering only for the local player.

I think this is important tho (too lazy to actually search where tf i used this event before and changing it if needed)

**Comment by Lpsd** [comment]
I think we discussed using `onClientPedWeaponSwitch` in #development on Discord

**Comment by qaisjp** [comment]
workflow needs to be restarted when mta website is not lagging balls

**Review by qaisjp — CHANGES_REQUESTED** [review]
`onClientPedWeaponSwitch` sounds good https://github.com/multitheftauto/mtasa-blue/pull/1143#issuecomment-549091031

**Comment by StrixG** [comment]
The branch was deleted.

**Comment by Woovie** [comment]
I started to recover this. Waiting to hear back from @qaisjp on the correct way to have done this, cuz I don't think I did.

https://github.com/Woovie/mtasa-blue/tree/onClientPlayerWeaponSwitch-remote

**Pull request closed without merging**